.github/ISSUE_TEMPLATE/bug-report-playback.yaml

```diff
@@ -20,10 +36,25 @@ body:
         Instead, I expect …
     validations:
       required: true
+  - type: textarea
+    id: mediainfo
+    attributes:
+      label: Media info of the file
+      description: |
+        Please share the media information for the file causing issues. You can use a variety of tools to retrieve this information.
+        - Use ffprobe (`ffprobe ./file.mp4`)
+        - Use the Media Info tool (set to text format, download here: https://mediaarea.net/en/MediaInfo)
```

Comment:
I think we can remove this one from the list, I'm don't think advising users to install some random software is that useful if they can already copy the media info from the webui/ffprobe